### ISSUE TITLE
Mekanism cardboard box won't wrap and pick up inventories

### ISSUE THREAD

**Issue by boballama** [FTBTeam/FTB-Presents-Direwolf20-1.16#41]
Latest DW20-1.16 modpack, version 1.1.2, Mekanism cardboard box does not wrap and pick up any inventories--including vanilla chests.  Cardboard box cannot be placed until you backup a step then it just plops down on the ground in front of the chest or spawner I was trying to pick up.

I see no issues being reported anywhere.

**Comment by LWmods** [comment]
this happens when you have something in your offhand, have you tried with nothing there ?

**Comment by boballama** [comment]
There is nothing in my offhand.  I tried picking up a chest with and empty offhand and again with torches in my offhand.  No difference.  Shift, right click (sneak) does not wrap the chest with the cardboard box.

**Comment by LWmods** [comment]
you don't have to sneak when placing the box on a chest etc just right click while holding it should do it.

**Comment by boballama** [comment]
If you don't sneak the inventory opens instead of getting wrapped.

**Comment by boballama** [comment]
I got the cardboard box to work.  First, your offhand only needs to be empty to unwrap the box, not wrap it.  I saved a copy of my saved game, deleted the modpack, removed all the files in .localCache that had creation dates similar to my DW20 install, reloaded the pack, started the game, exited, copied my previous save back to the saves folder, and re-entered my original world.  Cardboard box still didn't work.  I made a new cardboard box and this one worked.  So I tossed my original 3 cardboard boxes into a cactus and am now using newly-created ones.  Possibly some glitch between the original cardboard box that was made in version 1.1.0 and the 1.1.2 update.  I am closing this issue.
